roadmap/implementers-guide/src/types/runtime.md

```diff
@@ -41,7 +41,7 @@ struct HostConfiguration {
 	/// The amount of sessions to keep for disputes.
 	pub dispute_period: SessionIndex,
 	/// How long after dispute conclusion to accept statements.
-	pub dispute_post_conclusion_acceptance_period: BlockNumber,
+	pub dispute_post_conclusion_code_retention_period: BlockNumber,
```

Comment:
this was renamed in the guide, but not in the code

Reply:
nice catch